src/docs/product/integrations/cloud-monitoring/aws-lambda/index.mdx

```diff
@@ -94,31 +94,35 @@ In the table view, you will also see the version of the Lambda layer below the n
 
 ![](configure.png)
 
+## Verify
+
+Run your AWS Lambda function. Performance data will be sent to Sentry and will show up in the "Performance" section of Sentry. If your function raises an error it will show up in the "Issues" section of Sentry.
+
 ## Remove Installation
 
 Note that when the integration is uninstalled, your Lambda functions will still be instrumented with Sentry and you will continue to receive errors. If you want to disable Sentry for all your Lambda functions, you should disable Sentry for each function before uninstalling. Note, you can also uninstall and then reinstall if you would like to change the Sentry project for your Lambdas.
 
 Learn more about [how this integration works](/product/integrations/cloud-monitoring/aws-lambda/how-it-works/).
 
-## Serverless Framework
+## Notes on Specific Frameworks
+
+### Serverless Framework
 
-Note that this integration will configure your existing lambda functions correctly to use the SDK layer and set the appropriate environment variables as per [how this integration works](/product/integrations/cloud-monitoring/aws-lambda/how-it-works/). You can check the output `aws lambda get-function-configuration --function-name <yourFunctionName>` and this works.
+If you are using [Serverless Framekwork](https://www.serverless.com/) note that this integration will configure your existing lambda functions correctly to use the SDK layer and set the appropriate environment variables as per [how this integration works](/product/integrations/cloud-monitoring/aws-lambda/how-it-works/). You can check the output `aws lambda get-function-configuration --function-name <yourFunctionName>` and this works.
```

Comment:
```suggestion
If you're using a [serverless framework](https://www.serverless.com/), note that this integration will configure your existing Lambda functions correctly to use the SDK layer and set the appropriate environment variables. Learn more about it [here](/product/integrations/cloud-monitoring/aws-lambda/how-it-works/). 

You can confirm that the output works by inputting `aws lambda get-function-configuration --function-name <yourFunctionName>`. However, when you perform a subsequent `sls deploy` the environment variables aren't maintained even though it looks like the reference to the layer is retained. 

You can set the layer definition and environment variables in `serverless.yml` as a workaround, but there are two problems with this: you have to manually add the reference to the SDK layer for every function you want "Sentrified" and you can't maintain the latest layer version automatically.
```
It seems a little odd to end on a workaround that has known problems. Is there anything else we can add here (like any other workarounds or which specific use cases this work around would be or wouldn't be good for?)